proto/internal/temporal/server/api/historyservice/v1/service.proto

```diff
@@ -276,6 +276,10 @@ service HistoryService {
     rpc RefreshWorkflowTasks(RefreshWorkflowTasksRequest) returns (RefreshWorkflowTasksResponse) {
     }
 
+    // UnblockWorkflowExecution immediately unblocks a blocked/delayed workflow.
+    rpc UnblockWorkflowExecution(UnblockWorkflowExecutionRequest) returns (UnblockWorkflowExecutionResponse) {
```

Comment:
There's already a ScheduleWorkflowTask rpc defined for history service, let's see if that can be reused. May need to introduce some new fields in the request.